demo-ng8/package.json

```diff
@@ -47,6 +47,6 @@
     "protractor": "~5.4.2",
     "ts-node": "~8.3.0",
     "tslint": "~5.18.0",
-    "typescript": "^3.4.5"
+    "typescript": "~3.4.5"
```

Comment:
Does this change matter?

Reply:
Appreciate the time and attention, will review and check this as soon as I'm able :+1:

Reply:
Yeah, Angular 8 doesn’t support the latest TS version.

Reply:
I don't think you introduced this problem though, just wondering since I've had to lock in for testing purposes to 3.4.5 explicitly

Reply:
Wait.. that's correct.

https://jubianchi.github.io/semver-check/#/^3.4.5/3.6.4
the latest 3.6.4 meets the requirement but is outside the accepted range.

Needed to stay as ~3.4.5 to keep it safe for the build
https://jubianchi.github.io/semver-check/#/~3.4.5/3.6.4

I think...

Reply:
Yeah that’s correct